### PR TITLE
Mckale Chung Computer Vision Bootcamp Pull Req

### PR DESCRIPTION
**Pull request by cehune**
Hi! Please let me know if there's I should change or can improve on. Here's the images of the epoch statistics,  training and validation loss, validation accuracy, and final heatmap. At the top of the heatmap is the model accuracy, of 75%.

image
image
image
image

### CODE REVIEW

**Comment by Trotyl15** [comment]
Sorry for the late reply, but it looks good! You can check out the onboarding document [here](https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2060353537/CV+Onboarding). Welcome to the team!

**Comment by cehune** [comment]
Awesome, thanks!!
